### PR TITLE
Remove multiuser module from single-user integration tests

### PR DESCRIPTION
**Pull request by mshaposhnik**
### What does this PR do?
Remove multiuser  signature key dependency from single-user integrationTCK  tests set.

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

**Review by skabashnyuk — APPROVED** [review]

**Comment by mshaposhnik** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/855/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/855//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11355
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by mshaposhnik** [comment]
ci-test

File: wsmaster/integration-tests/postgresql-tck/pom.xml [hunk 1]
```diff
@@ -49,11 +49,6 @@
             <groupId>org.eclipse.che.infrastructure</groupId>
             <artifactId>infrastructure-kubernetes</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.eclipse.che.multiuser</groupId>
-            <artifactId>che-multiuser-machine-authentication</artifactId>
```
Comment (sleshchenko) [inline wsmaster/integration-tests/postgresql-tck/pom.xml:54]: Should not be the following lines be removed as well?
https://github.com/eclipse/che/blob/c6f0751c740bb4397aa04d9a62b4000596c8794a/wsmaster/integration-tests/postgresql-tck/pom.xml#L167-L171

**Review by sleshchenko — APPROVED** [review]

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/856/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/856//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11355
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Pull request merged**